plugins/inputs/exec/exec.go

```diff
@@ -70,9 +70,16 @@ func (c CommandRunner) Run(
 	command string,
 	timeout time.Duration,
 ) ([]byte, []byte, error) {
-	split_cmd, err := shellquote.Split(command)
-	if err != nil || len(split_cmd) == 0 {
-		return nil, nil, fmt.Errorf("exec: unable to parse command, %s", err)
+	var split_cmd []string
+	var err error
+	//cannot use shellquote.Split on Windows as it threats Windows path separator char '\' as a mark for backslashed special char
```

Comment:
I don't understand.  
When I had discovered this problem I was wondering how can be that exec plugin actually works for anyone on Windows. Only condition it can work now is that there is no path in the command.
How this fix would break existing config files?